src/react-queries/hooks/useAllowance.ts

```diff
@@ -6,15 +6,15 @@ import { selectAllowanceCryptoBaseUnit } from './selectors'
 type UseAllowanceArgs = {
   assetId: string | undefined
   spender: string | undefined
-  from: string | undefined
+  pubKey: string | undefined
```

Comment:
pref to keep allowance args named akin to contract call args imo

Reply:
Renamed back to `from` in d681485bd9